File: 9-ucinki-in-cistost/predavanja/vsota.py
def ustrezen(n):
    return n % 3 == 0 or n % 5 == 0

for _ in range(100):
    razpon = range(1, 100000)
    veckratniki = [n for n in razpon if ustrezen(n)]
    dolzina = len(veckratniki)

File: 9-ucinki-in-cistost/predavanja/vsota.ml
let rec range a b =
  if a >= b then [] else a :: range (succ a) b
in

let ustrezen n =
  n mod 3 = 0 || n mod 5 = 0
in

for _ = 1 to 100 do 
  range 1 100000
  |> List.filter ustrezen
  |> List.length
  |> ignore
done
